docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -83,6 +83,9 @@ Configuration properties
       - Disables using the address published by Elasticsearch to connect for
         queries.
       -
+    * - ``elasticsearch.legacy-pass-through-query-enabled``
```

Comment:
You are using `elasticsearch.legacy-pass-through-query.enabled` in code. Please update.